app/models/shift_survey.rb

```diff
@@ -0,0 +1,21 @@
+class ShiftSurvey < ApplicationRecord
+  extend DateRangeFilterHelper
+
+  belongs_to :shift
+  after_create :generate_token
+  serialize :ratings, Array
```

Comment:
Instead of serializing the `text` column, what if we used a Postgres array column type? https://dev.to/kputra/rails-postgresql-array-1jn0